### PR TITLE
Landing version 2

### PR DESCRIPTION
**Pull request by rumana-aafrin**
### Implementation of the landing page

Occured some issue before.. I have updated the code for the landing page frontend again.

# How to test?
Run the client and see the result in localhost:3000/teams-check-in

# Checklist
[Yes] I have performed a self-review of my own code
[Yes ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
[Yes ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
[Yes ] This PR is ready to be merged and not breaking any other functionality

landingpage

### CODE REVIEW

File: packages/client/.eslintrc [hunk 1]
```diff
@@ -15,6 +15,12 @@
     "react/state-in-constructor": "off",
     "react/jsx-props-no-spreading": "off",
     "react/function-component-definition": "off"
+    // "import/no-extraneous-dependencies" : [
```
Comment (ddobby94) [inline packages/client/.eslintrc]: Please remove these commented out lines

File: packages/client/public/index.html [hunk 2]
```diff
@@ -10,34 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
```
Comment (ddobby94) [inline packages/client/public/index.html:13]: This was generated by React.js itself, so no need to modify this file 😄

File: packages/client/src/components/Login/Testimonials.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    image: rumana,
+    text: 'Being able to see past check-ins helps us track our progress and stay accountable.',
+    author: 'Rumana, Designer',
+  },
+];
+
+const Testimonial = ({ image, text, author }) => {
+  return (
+    <div className="testimonial-card">
+      <img className="images" src={image} alt={author} />
```
Comment (ddobby94) [inline packages/client/src/components/Login/Testimonials.jsx:32]: Nice catch with the `alt` tag! It is always required to have a description for the non-visual users :)

File: packages/client/src/containers/LandingPage/LandingPage.Container.js [hunk 4]
```diff
@@ -1,25 +1,85 @@
-import React, { useEffect, useState } from 'react';
-import { apiURL } from '../../apiURL';
-import './LandingPage.Style.css';
+// import React, { useEffect, useState } from 'react';
```
Comment (ddobby94) [inline packages/client/src/containers/LandingPage/LandingPage.Container.js]: Please remove this commented out part

File: packages/client/src/containers/LandingPage/LandingPage.Container.js [hunk 5]
```diff
@@ -1,25 +1,85 @@
...
+
+import React from 'react';
+import Testimonials from '../../components/Login/Testimonials';
+import logo from './logo.svg';
+import { Typography, TextField, Button } from '@mui/material';
+import styled from '@emotion/styled';
+import './LandingPage.Style.css';
+// Import necessary components from Material-UI
 
+const CustomButton = styled(Button)({
```
Comment (ddobby94) [inline packages/client/src/containers/LandingPage/LandingPage.Container.js]: It is always good to have styled components, however, not many big companies will be using this library, because it is very difficult to mantain.
As simple as when I am trying to edit the style of an element, I usually go to the css files. If some of them will have styles in the `.jsx` files it will create some confusions.

What I would suggest is to create your own styled button, mixing Material UI and css, if you want to override the style every time:
```
// src/custom-components/Button/Button.jsx

Button.propTypes = {
  onClick: PropTypes.function.isRequired,
  variant: PropTypes.string.isRequired,
  // etc.
};

export const Button = ({ onClick, variant, children }) => {
 
  return (
    <Button onClick={onClick} className={`button-${variant}`}>{children}</Button>
  );
};

// And the css next to it:
// src/custom-components/Button/button.css

.button-primary {
  // custom-css
}

.button-secondary {
  // ...
}
```

If it is a one time style, only applied here you can just simple add the `className="custom-css"` to the material UI component.

File: packages/client/src/containers/LandingPage/LandingPage.Container.js [hunk 6]
```diff
@@ -1,25 +1,85 @@
...
-        <div key={example.id}>{example.title}</div>
-      ))}
+    <div className="login-page">
+      <div className="testimonial-container">
+        <Typography className="text-field" variant="h4" color="white">
+          Don’t forget to invite teammates!
+        </Typography>
+        <div className="testimonial-section">
+          <div className="testimonial-cards">
+            <Testimonials /> {/* Render the Testimonials component */}
```
Comment (ddobby94) [inline packages/client/src/containers/LandingPage/LandingPage.Container.js]: This one is easy to understand, no need for clarification comment :)

File: packages/client/src/containers/LandingPage/LandingPage.Container.js [hunk 7]
```diff
@@ -1,25 +1,85 @@
...
+        <div className="login-details">
+          <Typography variant="h5" fontWeight="bold">
+            Log in to Teams Check-in
+          </Typography>
+          <div className="login-action">
+            <div className="login-text">
+              <Typography className="text-field" variant="body1">
+                Enter your team code
+              </Typography>
+              <TextField
```
Comment (ddobby94) [inline packages/client/src/containers/LandingPage/LandingPage.Container.js]: Next up is to create a function and a variable to catch what the user wrote :)
You can use a console.log to print out the user's input! But this can be a separate PR

**Review by ddobby94 — COMMENTED** [review]
Great job, left a few comments, let me know if something is not clear!

File: packages/client/public/index.html (hunk 2 above)
Comment (rumana-aafrin) [reply packages/client/public/index.html:13]: You're right! I have deleted some files in React.js, because it's filled with irrelevant contents. And we used to remove in our class as well. That's the reason behind!

File: packages/client/src/containers/LandingPage/LandingPage.Container.js (hunk 7 above)
Comment (rumana-aafrin) [reply packages/client/src/containers/LandingPage/LandingPage.Container.js]: Sure! I will create a next PR to integrate the backend to catch the user input!

**Review by ddobby94 — APPROVED** [review]
Keep up the good work! :)
